wordle-web/app.vue

```diff
@@ -1,17 +1,43 @@
 <template>
   <NuxtLayout>
     <v-app>
-      <v-app-bar color="primary" :elevation="2">
+      <v-app-bar color="primary">
+
+        <v-app-bar-title>Pentagram</v-app-bar-title>
         <template v-slot:prepend>
-          <v-icon color="secondary"> mdi-book </v-icon>
+          <v-btn icon @click="router.push('/')">
+            <v-icon size="x-large">mdi-pentagram</v-icon>
+          </v-btn>
         </template>
-
-        <v-app-bar-title>WordleApp</v-app-bar-title>
-
-        <v-btn @click="router.push('/')">Home</v-btn>
-        <v-btn @click="router.push('/test')">Test</v-btn>
+        <v-btn @click="showUserNameDialog = true">{{ userName }}</v-btn>
+        <UserNameDialog v-model="showUserNameDialog" />
```

Comment:
![keyboard handler issue](https://github.com/IntelliTect-Samples/EWU-CSCD379-2024-Spring/assets/114778753/08afd991-29fd-4b16-81dd-ba913329f30d)

An issue is that now whenever the user is prompted for their name is that it will record their inputs into the game as well due to the `keyUp` event. You could maybe try and disable the event when `showUserNameDialog` is true so that it won't record the inputs. We did something similar for our `ValidWords` component in Assignmenet2

Reply:
I think we've fixed this now, thanks for pointing it out!